src/gpg2john.c

```diff
@@ -1363,6 +1363,74 @@ private void
 	Private_Packet,
 };
 
+private void
+(*tag_func4[])(int, int, int, char *) = {
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	Symmetrically_Encrypted_Data_Packet,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	NULL,
+	Symmetrically_Encrypted_and_MDC_Packet,
```

Comment:
The functions removed from the old/first array are separated by 8 other pointers, but here they're separated by only 7 NULLs. A bug?

Separately (but maybe related), I notice that the old array initializer had 65 lines, whereas the tag mask is such that there should probably be only 64 - a bug? Harmless?

Reply:
The first commit (f4df7727) of gpg2john had that non-multiple of 4. Jim added a second typedef in 2fee9ee0c for #477.